packages/qdrant-loader-core/src/qdrant_loader_core/graph/__init__.py

```
@@ -0,0 +1,43 @@
+try:
+    from qdrant_loader.config import get_settings
+except ImportError:  # pragma: no cover
+    get_settings = None
+
+__all__ = ["FalkorGraphStore", "GraphNode", "GraphEdge", "GraphStore", "SubGraph"]
+
+
+_graph_store = None
+
+
+async def get_graph_store():
```

Ruff (0.15.14)

[warning] Missing return type annotation for private function `get_graph_store`

(ANN202)